### PR TITLE
Fix curl command line options in the json rpc md file

### PR DESCRIPTION
**Pull request by platfowner**

### CODE REVIEW

File: JSON_RPC_API.md [hunk 1]
```diff
@@ -51,31 +51,244 @@ Returns the value, write rule, owner rule, or function at the given path in the
...
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": {
+    "result": {
+      "token": {
+        "name": "AI Network",
+        "symbol": "AIN",
+        "total_supply": 700000000,
+        "bridge": {
```
Comment (jiyoung-an) [inline JSON_RPC_API.md:167]: what does the "bridge" and fields below mean?

File: JSON_RPC_API.md [hunk 2]
```diff
@@ -51,31 +51,244 @@ Returns the value, write rule, owner rule, or function at the given path in the
...
+            }
+          }
+        }
+      },
+      "consensus": {
+        "min_stake_for_proposer": 10000000,
+        "max_stake_for_proposer": 10000000,
+        "min_num_validators": 5,
+        "max_num_validators": 20,
+        "genesis_proposer_whitelist": {
```
Comment (jiyoung-an) [inline JSON_RPC_API.md:188]: what is "proposer"?

File: JSON_RPC_API.md (hunk 1 above)
Comment (platfowner) [reply JSON_RPC_API.md:167]: a bridge is a service for exchanging tokens. here it exchanges AINs for ETHs and vice versa with a fixed rate.

File: JSON_RPC_API.md (hunk 2 above)
Comment (platfowner) [reply JSON_RPC_API.md:188]: proposers mean block producer blockchain nodes. we allow only whitelisted blockchain nodes to propose new blocks.

File: JSON_RPC_API.md (hunk 1 above)
Comment (jiyoung-an) [reply JSON_RPC_API.md:167]: oh bridge server is included in the blockchain..! thank you for letting me know

File: JSON_RPC_API.md (hunk 2 above)
Comment (jiyoung-an) [reply JSON_RPC_API.md:188]: oh I see, thank you for letting me know!

**Pull request merged**